projects/fsastorefrontlib/src/cms-components/comparison-table/comparison-table-sync-pilot/comparison-table-sync-pilot.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Observable, Subscription } from 'rxjs';
+import { CmsService, User, WindowRef } from '@spartacus/core';
+import { UserAccountFacade } from '@spartacus/user/account/root';
+import { CMSConnectionComponent } from '../../../occ/occ-models/cms-component.models';
+import { SyncPilotService } from '../../sync-pilot/sync-pilot.service';
+
+@Component({
+  selector: 'cx-fs-comparison-table-sync-pilot',
+  templateUrl: './comparison-table-sync-pilot.component.html',
+})
+export class ComparisonTableSyncPilotComponent implements OnInit, OnDestroy {
```

Comment:
Shouldn't this component extend the generic one `SyncPilotConnectionComponent `?

Reply:
This was the product of trying to resolve problems with multiple components using the same connection URL. Maybe we'll revert this back to extending component, but we can discuss this after SyncPilot guys fix the problem with single connection. I think that we may not need one generic component, but a service, like it is now. Those methods in service can perhaps be used by any CMS connection component. Not sure which approach is more generic...

Reply:
I think it is ok to have generic component because `setConfig`, `connect`, `redirectToAgent` and `abort` is something that all components need.